source/DD4T.Templates.Base/Builder/TargetGroupBuilder.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using System.Collections.Generic;
 using DD4T.ContentModel;
-using DD4T.Templates.Base.Utils;
 using Tridion.ContentManager.AudienceManagement;
 using Condition = DD4T.ContentModel.Condition;
 using ConditionOperator = DD4T.ContentModel.ConditionOperator;
 using CustomerCharacteristicCondition = Tridion.ContentManager.AudienceManagement.CustomerCharacteristicCondition;
 using Dynamic = DD4T.ContentModel;
 using NumericalConditionOperator = DD4T.ContentModel.NumericalConditionOperator;
+using AudienceManagement = Tridion.ContentManager.AudienceManagement;
```

Comment:
There are now 4 references to Tridion.ContentManager.AudienceManagement? Can't you reuse one?